src/components/POLine/hooks/useFundsById/useFundsById.js

```diff
@@ -0,0 +1,49 @@
+import { useQuery } from 'react-query';
+
+import {
+  useOkapiKy,
+  useNamespace,
+} from '@folio/stripes/core';
+import {
+  batchRequest,
+  FUNDS_API,
+} from '@folio/stripes-acq-components';
+
+export const useFundsById = (fundIds = [], options) => {
```

Comment:
default params should be last: makes sense to set default options' value as `{}`